src/components/toast-notification-manager/daikin-toast-notification-manager.ts

```diff
@@ -0,0 +1,233 @@
+        "top-left": ["items-start", "top-4", "left-4"],
+        "top-right": ["items-end", "top-4", "right-4"],
+        bottom: ["items-center", "m-auto", "bottom-4", "left-0", "right-0"],
+        "bottom-left": ["items-start", "bottom-4", "left-4"],
+        "bottom-right": ["items-end", "bottom-4", "right-4"],
+      },
+    },
+  }
+);
+
+export type ToastPosition = MergeVariantProps<typeof cvaContainer>["position"];
+
+/**
+ * The toast manager component manages the position and display state of notification toasts.
+ * Just place a notification toast in the slot and it will automatically be placed, stacked and animated.
+ *
+ * @fires close - A custom event emitted when a user clicks the close button. Even if this is fired, the toast inside the slot will not be deleted from the component side, so you will need to delete it yourself.
+ *
+ * @slot A slot for toasts. Place `daikin-toast-notification` elements here.
+ *
+ * @example
+ *
+ * ```js
+ * import "@daikin-oss/design-system-web-components/components/toast-notification-manager/index.js";
+ * import "@daikin-oss/design-system-web-components/components/toast-notification/index.js";
+ * ```
+ *
+ * ```html
+ * <daikin-toast-notification-manager>
+ *   <daikin-toast-notification name="toast1">
+ *     <span slot="title">Toast title</span>
+ *     <span slot="description">Toast description</span>
+ *   </daikin-toast-notification>
+ *   <daikin-toast-notification name="toast2">
+ *     <span slot="title">Toast title</span>
+ *     <span slot="description">Toast description</span>
+ *   </daikin-toast-notification>
+ *   <daikin-toast-notification name="toast3">
+ *     <span slot="title">Toast title</span>
+ *     <span slot="description">Toast description</span>
+ *   </daikin-toast-notification>
+ * </daikin-toast-notification-manager>
+ * ```
+ */
+@customElement("daikin-toast-notification-manager")
+export class DaikinToastNotificationManager extends LitElement {
+  static override readonly styles = css`
+    ${unsafeCSS(tailwindStyles)}
+
+    :host {
+      display: block;
+    }
+  `;
+
+  /**
+   * Specify where on the screen the toast will be displayed.
+   */
+  @property({ type: String, reflect: true })
+  position: ToastPosition = "bottom-right";
+
+  /**
+   * Specify how many seconds after the toast is displayed the close event should be fired (unit = ms).
+   * If `null`, there will be no automatic close event firing.
+   */
+  @property({ type: Number, reflect: true })
+  duration: number | null = null;
+
+  @queryAssignedElements({ selector: "daikin-toast-notification" })
+  private readonly _items!: readonly DaikinToastNotification[];
+
+  private _knownItemSet: ReadonlySet<DaikinToastNotification> = new Set();
+
+  private _containerRef = createRef<HTMLElement>();
+
+  get _positionY(): "top" | "bottom" {
+    return this.position.startsWith("top") ? "top" : "bottom";
+  }
+
+  private _close(target: DaikinToastNotification) {
```

Comment:
Prevent `_close` called more than once.

```suggestion
  private _close(target: DaikinToastNotification) {
    if (this._closedItemSet.has(target)) {
      return;
    }
    this._closedItemSet.add(target);

```